content/md/products-super-power/work-on-your-assets.md

```diff
@@ -32,6 +33,30 @@ You can find a **locale and channel switcher** on each asset, meaning that if yo
 
 ![Assets Locale Switcher](../img/Assets_LocaleSwitcher.png)
 
+## Bulk edit multiple assets
+If you have the `edit` permission on an asset family, it is possible to bulk edit your own selection of assets within this asset family.
+Let's imagine that you need to add the same tag to several assets, for instance, their main color attribute. Instead of manually adding the tag for each of these assets, you can easily do it in one action for your selection of assets, thanks to a bulk edition. Here's how to do so:
```

Comment:
Hi @felix-couprie your example here is to add a tag that would contain the main color of the image right? Otherwise, can we be more explicit with the use case? "main color attribute" could be misleading.
Example given below

Reply:
```suggestion
Let's imagine you need to add a tag to several assets from the same family, this tag will give information on the main color of the image, and this color is the same for all your selected assets. Instead of manually adding the tag for each of these assets, you can easily do it in one action for your selection of assets, thanks to a bulk edition. Here's how to do so:
```

Reply:
Thanks Laure, it sounds clearer!